source/_docs/guides/lando.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Install and Configure Lando
+description: Install and Configure Lando for local development of WordPress sites.
+contributors: [alexvasquez]
+type: guide
+permalink: docs/guides/:basename/
+---
+
+This guide will help you get up and running with [Lando](https://thinktandem.io/lando/){.external}, an Open Source development tool intended for developers who prefer a painless, easily customizable local server environment configuration they can track in their source control repositories.
+
+<div class="alert alert-info" role="alert" markdown="1">
+#### Note {.info}
+Lando does not currently provide a Graphical User Interface (GUI). Everything is managed from the command line. 
+</div>
+
+## Before You Begin
+
+Be sure that you have:
+
+- An existing WordPress site on Pantheon, or [create](https://dashboard.pantheon.io/sites/create){.external} one.
+- Having a working knowledge of local server environments is helpful but not entirely necessary.
+- While you’re at it be sure to check out [Lando getting started guide](https://docs.devwithlando.io/dev/started.html){.external} and [Lando CLI usage](NEED-LINK).
+
+
+Installing Lando is straight forward. Be sure to check its [system requirements](https://docs.devwithlando.io/installation/system-requirements.html){.external} to ensure you can proceed.
+
+## Downloading and Install Lando
+
+- The latest release of Lando for your OS can be found [here](https://github.com/lando/lando/releases){.external}. Download the appropriate release for your Operating System.
+- Click to open the installer package. The installer has a few pre-flight checks it runs before getting started.
+
+![Image of the Lando installer](/source/docs/assets/images/guides/lando/IMAGE.png "Lando Installer")
```

Comment:
@digisavvy I couldn't access the Google URLs. Add your images to `/source/docs/assets/images/guides/lando/`